pinecone/core/client/api/index_operations_api.py

```diff
@@ -134,7 +134,7 @@ def configure_index_with_http_info(
         :rtype: tuple(IndexMeta, status_code(int), headers(HTTPHeaderDict))
         """
 
-        _hosts = ["https://controller.{environment}.pinecone.io"]
+        _hosts = ["https://api.pinecone.io"]
```

Comment:
Changes in this file are generated, derived from changes to the `servers` field in our openapi spec stored in the other repository.